### PR TITLE
Bug Fixes

### PR DESCRIPTION
**Pull request by kevin-10**
- Wraith Form leech consolidated in AOS.cs Damage method
- fixed loot props on jewelry, such as FC2, eaters and casting Focus. existing items will be nerfed
- added proper itemid to Powder of Fortification Keg
- Added Deer Mask, Bear Mask, Orc Mask and Tribal masks to tailoring menu. They will also be able to be imbued
- Fixed cliloc in Tailoring menu
- Possible fix for gold leaking to internal map
- Blackthorn clothing items no longer spawn with durability. Existing items have been fixed
bb27362

### CODE REVIEW

File: Scripts/Services/Revamped Dungeons/BlackthornDungeon/Items/RoyalBritannianBase/TunicBearingTheCrestOfBlackthorn.cs [hunk 1]
```diff
@@ -19,21 +19,6 @@ public TunicBearingTheCrestOfBlackthorn()
             Hue = 0xe8;
         }
 
-        public override int InitMinHits
```
Comment (Argalep) [inline Scripts/Services/Revamped Dungeons/BlackthornDungeon/Items/RoyalBritannianBase/TunicBearingTheCrestOfBlackthorn.cs]: Wrong

image

File: Scripts/Services/Revamped Dungeons/BlackthornDungeon/Items/RoyalBritannianBase/SurcoatBearingTheCrestOfBlackthorn.cs [hunk 2]
```diff
@@ -19,21 +19,6 @@ public SurcoatBearingTheCrestOfBlackthorn()
             Hue = 0xe8;
         }
 
-        public override int InitMinHits
```
Comment (Argalep) [inline Scripts/Services/Revamped Dungeons/BlackthornDungeon/Items/RoyalBritannianBase/SurcoatBearingTheCrestOfBlackthorn.cs]: wrong 

image

File: Scripts/Services/Revamped Dungeons/BlackthornDungeon/Items/RoyalBritannianBase/GargishSashBearingTheCrestOfBlackthorn.cs [hunk 3]
```diff
@@ -19,21 +19,6 @@ public GargishSashBearingTheCrestOfBlackthorn()
             Hue = 0xe8;
         }
 
-        public override int InitMinHits
```
Comment (Argalep) [inline Scripts/Services/Revamped Dungeons/BlackthornDungeon/Items/RoyalBritannianBase/GargishSashBearingTheCrestOfBlackthorn.cs]: wrong

image

File: Scripts/Services/Revamped Dungeons/BlackthornDungeon/Items/RoyalBritannianBase/DoubletBearingTheCrestOfBlackthorn.cs [hunk 4]
```diff
@@ -19,21 +19,6 @@ public DoubletBearingTheCrestOfBlackthorn()
             Hue = 0xe8;
         }
 
-        public override int InitMinHits
```
Comment (Argalep) [inline Scripts/Services/Revamped Dungeons/BlackthornDungeon/Items/RoyalBritannianBase/DoubletBearingTheCrestOfBlackthorn.cs]: wrong

image

**Comment by Tasanar** [comment]
Seems just the sash layer items?

**Comment by kevin-10** [comment]
Good catch!

**Comment by kevin-10** [comment]
Argalep, approve and merge. You're going to have to close and resend your PR. It has some conflicts.

**Pull request merged**